pkg/virt-controller/watch/vmi.go

```diff
@@ -2237,6 +2237,14 @@ func (c *VMIController) updateInterfaceStatus(vmi *virtv1.VirtualMachineInstance
 			status.PodConfigDone = true
 		}
 		newStatus = append(newStatus, status)
+		// after this loop, the only statuses on `indexedVMIStatus` would be guest agent related
```

Comment:
The comment is not accurate. You may also have interfaces that were added directly to libvirt (by using virt for example).